tutorials/Python/12_ProteinAnalysis/12_ProteinAnalysis.ipynb

```diff
@@ -0,0 +1,468 @@
+    "\n",
+    "How many of them are waters?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a selection from non_protein_residues that only includes waters\n",
+    "# In the PDB waters are named HOH, so we can make a selection from this\n",
+    "# Here we use the \"resname\" selection to select by residue name\n",
+    "waters = non_protein_residues.select_atoms('resname HOH')\n",
+    "print(\"number of waters: \", waters.atoms.n_residues)\n",
+    "\n",
+    "# What about non-water non-protein residues?\n",
+    "not_water = non_protein_residues.select_atoms('not resname HOH')\n",
+    "print(\"number of non-water, non-protein residues: \", not_water.atoms.n_residues)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we can see, there is 1 non-protein non-water residue.\n",
+    "Let's find out more information about it.\n",
+    "\n",
+    "First let's see what this residue is called. Here we will be using the `residues` object, which is like `atoms`, but rather than containing atomic information it contains information about the residues. Specifically here we are looking at `resnames` that tells us what the residue name is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"residue name: \", not_water.residues.resnames)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "MK1 is the PDB name for the drug indinavir. You can look at the PDB entry for it [here](https://www.rcsb.org/ligand/MK1).\n",
+    "\n",
+    "Since the PDB file contains per-atom information (in `atoms`), we can use MDAnalysis to list the atoms that make indinavir:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(not_water.atoms.types)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use the coordinates from the PDB file to obtain more information about the MK1 residue. Since MDAnalysis takes the coordinate information from the PDB file, we could use the MK1 coordinates (acessible under `not_water.atoms.positions`) to calculate the center of geometry. MDAnalysis provides a simple method for doing this called `center_of_geometry()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(not_water.center_of_geometry())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Exercise 1 - Protein center of geometry\n",
+    "\n",
+    "What about the center of the geometry of the protein? Using the `protein_residues` subselection we made earlier, can you apply the same thing to work out what the center of geometry of the protein atoms is?"
```

Comment:
```suggestion
    "What about the center of the geometry of the protein? Using the `protein_residues` subselection we made earlier, apply the same thing to work out what the center of geometry of the protein atoms is."
```

I have the feeling that one would feel bad if we ask "Can you...?" and it does not manage to do it. I would just tell what needs to be done.

Reply:
Very good point.